autoload/asyncomplete/sources/gocode.vim

```diff
@@ -37,7 +37,7 @@ function! s:on_exec_events(info, id, data, event) abort
 endfunction
 
 function! s:write_buffer_to_tempfile(ctx) abort
-    let l:buf = getline(a:ctx['bufnr'], '$')
```

Comment:
we need to use `getbufline` instead.

Reply:
@prabirshrestha done, I think that's what you mean.